app/javascript/components/product/ProductDescription.tsx

```
@@ -0,0 +1,23 @@
+  description: string;
+}
+
+export function ProductDescription({ description }: Props) {
+  const html = renderMarkdown(description);
+
+  return (
+    <section className="border-t border-gray-200 pt-8">
+      <h2 className="text-xl font-bold text-gray-900 mb-6">Product Description</h2>
+      <div
+        className="prose prose-gray max-w-none prose-headings:text-gray-900 prose-headings:font-semibold prose-a:text-indigo-600 prose-code:text-indigo-600 prose-pre:bg-gray-900 prose-pre:text-gray-100"
+        dangerouslySetInnerHTML={{ __html: html }}
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)